authorization-common/src/main/resources/legacyRights.json

```diff
@@ -78,5 +78,6 @@
   "datahub/read": "datahub/read",
   "datahub/write": "datahub/write",
   "structured_events/read": "structured_events/read",
-  "iam/listAssignedResourceRoles": "iam/listAssignedResourceRoles"
+  "iam/listAssignedResourceRoles": "iam/listAssignedResourceRoles",
+  "environments/getAuditEvents": "environments/describeEnvironment"
```

Comment:
I couldnt see the _environments/getAuditEvents_ in UMS (in thunderhead repo). And I couldnt see the role-right assigment either. Please check the javadoc of PublicCloudBuiltInRolesProviderImpl class.

It seems to me this is not a legacy right: environments/describeEnvironment.